legacy-pages/js/import/wallet-import.js

```diff
@@ -1,6 +1,8 @@
 (function() {
     //Save the javascript wallet to the remote server
     function reallyInsertWallet(guid, sharedKey, password, successcallback) {
+        // Executing google recaptcha
+        initCaptcha()
```

Comment:
This is happening here because of expiration, makes sense to get the captcha validation once the file was added into the page to avoid expiration errors.